### DISCUSSION TITLE
Cloudflare代理求助

### DISCUSSION THREAD

**Discussion by heckerpowered** [XTLS/Xray-core#4187] [Q&A]
我现在正在使用Cloudflare Worker做代理，使用vless+websocket协议，worker的代码是网上白嫖的，不过看起来不支持xtls，加密方式也是none，类似多路复用和TCP快速打开的似乎也不支持，我想让代理支持XTLS，另外想请问使用Cloudflare代理时底层传输方式是不是只支持websocket

**Comment by richgk999** [comment]
最新的xhttp已经支持过CF了。
